modules/renter/hostdb/hostweight.go

```diff
@@ -322,12 +324,19 @@ func (hdb *HostDB) uptimeAdjustments(entry modules.HostDBEntry) float64 {
 			// Ignore the unsorted scan entry.
 			continue
 		}
+
+		blockDiff := scan.BlockHeight - recentBlockHeight
+		timePassed := float64(time.Duration(blockDiff) * 10 * time.Minute)
+		halftime := float64(1 * time.Hour * 24 * 30)
```

Comment:
halftime should be in `modules/renter/hostdb/consts.go` instead of being declared inline in 2 files.